LibTrixi.jl/src/api_jl.jl

```diff
@@ -49,3 +49,53 @@ function trixi_step_jl(simstate)
 
     return nothing
 end
+
+function trixi_ndims_jl(simstate)
+    mesh, _, _, _ = Trixi.mesh_equations_solver_cache(simstate.semi)
+    return ndims(mesh)
+end
+
+function trixi_nelements_jl(simstate)
+    _, _, solver, cache = Trixi.mesh_equations_solver_cache(simstate.semi)
+    return Trixi.nelements(solver, cache)
+end
+
+function trixi_nvariables_jl(simstate)
+    _, equations, _, api_jl = Trixi.mesh_equations_solver_cache(simstate.semi)
+    return Trixi.nvariables(equations)
+end
+
+function trixi_get_cell_averages_jl(data_, simstate)
+
+    mesh, equations, solver, cache = Trixi.mesh_equations_solver_cache(simstate.semi)
+    nelements = Trixi.nelements(solver, cache)
+    nvariables = Trixi.nvariables(equations)
+
+    # convert C to julia
+    data = unsafe_wrap(Array, data_, nelements*nvariables)
+
+    u_ode = simstate.integrator.u
+    u = Trixi.wrap_array(u_ode, mesh, equations, solver, cache)
+
+    for (index, element) in enumerate(Trixi.eachelement(solver, cache))
```

Comment:
I think `index` and `element` will always be the same here. `eachelement` just goes from 1 to `nelements`, and the same will happen for `index` (I guess)

Reply:
*bump*

Reply:
done now